src/peft/utils/loftq_utils.py

```diff
@@ -15,10 +15,16 @@
 # Reference code: https://github.com/yxli2123/LoftQ/blob/main/utils.py
 # Reference paper: https://arxiv.org/abs/2310.08659
 
+from __future__ import annotations
```

Comment:
Why this import?

Reply:
Yeah, I guess with the type annotations used in this file, it wouldn't be necessary, but I also don't think it hurts.